src/components/coblocks-deactivate-modal/test/modal.cypress.js

```diff
@@ -0,0 +1,51 @@
+describe( 'Component: CoBlocks Deactivate Modal', () => {
+	beforeEach( () => {
+		cy.intercept(
+			'GET',
+			'https://wpnux.godaddy.com/v3/api/feedback/coblocks-optout*',
```

Comment:
@sbalakrishnan1-godaddy Try replacing this line with the following, I think that will get e2e tests working on CircleCI:
```js
new RegExp( 'https://wpnux.godaddy.com/v3/api/feedback/coblocks-optout*' ),
```